op-challenger/game/keccak/types/types.go

```diff
@@ -21,6 +22,16 @@ type Leaf struct {
 	StateCommitment common.Hash
 }
 
+// Hash returns the hash of the leaf data. That is the
+// bytewise concatenation of the input, index, and state commitment.
+func (l *Leaf) Hash() common.Hash {
+	concatted := make([]byte, 0, 136+32+32)
```

Comment:
i am curious if this allocs at all due to the `...` and how it compares to using `copy`

Reply:
<img width="688" alt="Screenshot 2024-01-24 at 4 26 27 PM" src="https://github.com/ethereum-optimism/optimism/assets/21288394/4f76cc2c-343d-4768-b81a-081b52086268">

Reply:
> For example, `make([]int, 0, 10)` allocates an underlying array of size 10 and returns a slice of length 0 and capacity 10 that is backed by this underlying array.

Reply:
you can ignore this its more academic in nature bc what you did works, i have a personal preference for `copy` bc its more like c